feat(kmesh): UBI images (#3537)
Kong/docs.konghq.com: app/mesh/1.6.x/features/ubi-images.md

---
title: Red Hat Universal Base Images
---

> In addition to the standard Kong Mesh images build on Alipne Linux, Kong Mesh also ships with images based on [Red Hat Universal Base Image (UBI)](https://developers.redhat.com/products/rhel/ubi).

Reviewer: ```suggestion
In addition to the standard Kong Mesh images built on Alpine Linux, Kong Mesh also ships with images based on the [Red Hat Universal Base Image (UBI)](https://developers.redhat.com/products/rhel/ubi).
```

> Kong Mesh UBI images are distributed with all standard images, but with `ubi-` prefix.

Reviewer: ```suggestion
Kong Mesh UBI images are distributed with all standard images, but with the `ubi-` prefix:
```

* [kuma-dp UBI Image](https://hub.docker.com/r/kong/ubi-kuma-dp)
* [kuma-cp UBI Image](https://hub.docker.com/r/kong/ubi-kuma-cp)
* [kumactl UBI Image](https://hub.docker.com/r/kong/ubi-kumactl)
* [kuma-init UBI Image](https://hub.docker.com/r/kong/ubi-kuma-init)
* [kuma-prometheus-sd UBI Image](https://hub.docker.com/r/kong/ubi-kuma-prometheus-sd)

The base UBI variant for all images is `ubi-minimal`.

## Usage

> To use UBI images, we need to explicitly pass them when the control plane is installed

Reviewer: ```suggestion
To use UBI images, you need to explicitly pass them when the control plane is installed:
```
